### Paramteres used by the local planning node ###
local_planning_node:
  ros__parameters:

    # translation_[velocity, acceleration, jerk]_limits
    translation_constraints: [0.3, 0.15, 0.001]
    # rotation_[velocity, acceleration, jerk]_limits
    rotation_constraints: [0.3, 0.15, 0.001]
    # Radius of accepetance for waypoints switching
    radius_of_accepetance: 0.25
    # 
    dt: 0.1
    delta: 2.0
    kappa: 0.1


    

    
